src/Api/Controllers/AccountsController.cs

```diff
@@ -708,5 +713,33 @@ public async Task PutTaxInfo([FromBody]TaxInfoUpdateRequestModel model)
             };
             await _paymentService.SaveTaxInfoAsync(user, taxInfo);
         }
+
+        [HttpDelete("sso/{organizationId}")]
+        public async Task DeleteSsoUser(string organizationId)
+        {
+            var userId = _userService.GetProperUserId(User);
+            if (!userId.HasValue)
+            {
+                throw new NotFoundException();
+            }
+
+            var ssoUser = new SsoUser()
+            {
+                UserId = userId.Value,
+                OrganizationId = new Guid(organizationId),
+            };
+
+            await _organizationService.DeleteSsoUserAsync(ssoUser);
```

Comment:
I am not sure it is appropriate to pass a partially constructed ssouser for the purposes of just holding two parameters (userid and orgid). It may be more clear from the caller as to what it is tdoing to just pass those two parameters in. It can also lead to the implementation thinking this is a fully qualified sso user (with and id), which it is not.

Reply:
You could also look up the SsoUser by the two parameters and then pass it in. That way it is a fully contructed ssouser. You wouldn't need your new delete method then, since you could just call the regular delete by ssouser id.

Reply:
I'm not sure Option 2 would work without rolling a new SP or changing the current one. dbo.SsoUser_Delete runs off UserId and OrgId. I don't mind making that change, but that SP already existed so I was trying to work with what was there in creating the delete override.

Reply:
I passed in userId and orgId and removed the partial model creation